eframe/src/web/mod.rs

```diff
@@ -38,6 +39,7 @@ pub fn now_sec() -> f64 {
         / 1000.0
 }
 
+#[allow(dead_code)]
```

Comment:
remove dead code instead

Reply:
I thought it could be useful in future. but okay